src/writing-your-own-application/no-std-applications/index.md

```diff
@@ -0,0 +1,13 @@
+# Writing no_std applications
+
+The goal of this chapter is to provide a getting-started guide on using the Rust programming language with Espressif SoCs and modules using [esp-hal].
+
+Examples shown here usually apply to ESP32-C3 using the [ESP32-C3-DevKit-RUST-1] board.
+
+You can use any other ESP32, ESP32-C3, ESP32-S2, or ESP32-S3 development board but smaller code changes and configuration changes might be needed.
+
+Also, this section of the book will only cover working locally.
```

Comment:
I know what you mean here, but it may not be clear to beginners. While we would like to assume that people read the preceding chapters, it would probably be good to elaborate a bit on what is meant by this, even if it's just a few words.